docs/dev-setup/install.md

```diff
@@ -30,7 +30,8 @@ make peer
 
 To see what commands are available, simply execute the following commands:
 ```
-$ peer
+cd $GOPATH/src/github.com/hyperledger/fabric/build/bin
+./peer
 ```
```

Comment:
I should add that I have no problem with a note that mentions where the output goes.